.buildbot.sh

```diff
@@ -0,0 +1,41 @@
+#! /bin/sh
+
+set -eu
+
+rm -fr ./build && mkdir ./build 
+cd build
+
+export CARGO_HOME="${PWD}/.cargo"
+export RUSTUP_HOME="${PWD}/.rustup"
+
+# install rust
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs > rustup.sh
+sh rustup.sh --default-host x86_64-unknown-linux-gnu --default-toolchain stable -y --no-modify-path
+export PATH="${PWD}/.cargo/bin/:$PATH"
+
+# yk bulid
+git clone git@github.com:ykjit/yk.git 
```

Comment:
You'll need to use an `https` URL here, as the `git@` requires a key. Since you jsut need to check out master, I'd actually suggest:

```
git clone --depth=1 https://github.com/ykjit/yk
```

Reply:
ok will try

Reply:
Ah you'll need `--recurse-submodules` or similar too.